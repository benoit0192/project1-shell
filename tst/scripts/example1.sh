# example1.sh
# an example of valid shell script

# it can have comments like this

# and it can run commands, one per line
ls

uname

# unknown commands trigger error messages
bob
