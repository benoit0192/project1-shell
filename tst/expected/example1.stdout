/root/project1/src
Minix
